kumascript/macros/CSSAnimatedProperties.ejs

```diff
@@ -13,7 +13,7 @@ var animatedProps = [];
 // the animatedProps array.
 for (var property in data.properties) {
     if (data.properties.hasOwnProperty(property)) {
-        if (data.properties[property].animationType !== "discrete") {
+        if (data.properties[property].animationType !== "discrete" && data.properties[property].animationType !== "notAnimatable") {
```

Comment:
What about
```suggestion
        if (!["discrete", "notAnimatable"].includes(data.properties[property].animationType)) {
```
or anything to not make it such a long line with repetition. 

(note, I haven't actually tested it. just a style drive-by comment)

Reply:
One option would be to move that out to a variable (or function) and use that as a test in `if`.
Like: `const isAnimatable = ["discrete", "notAnimatable"].includes(data.properties[property].animationType)`.
Or

```js
const { animationType } = data.properties[property]
const isAnimatable = ["discrete", "notAnimatable"].includes(animationType)
if (!isAnimatable) { /* … */ }
```

Reply:
@peterbe This is exactly how I first intended to do it. But then I was confused by the usage of `var` in this script instead of `const` or `let`. So I just thought it should not use anything above ES5 and decided not to use `include` which is an ES2016 addition. That's why this long ugly line. Now I see I was wrong.

@Ryuno-Ki I think what @peterbe suggests is a shorter option, isn't it?